src/compas/datastructures/mesh/mesh.py

```diff
@@ -25,8 +25,10 @@
 from compas.geometry import subtract_vectors
 from compas.geometry import normal_polygon
 from compas.geometry import area_polygon
-from compas.geometry import bestfit_plane
-from compas.geometry import flatness
+
```

Comment:
are `bestfit_plane` and `flatness` not directly importable from `geometry`?

Reply:
it works when I use the function but the checks fail

Reply:
fixed a few imports. try again, i think it will work now...